src/SalmonDE/StatsPE.php

```diff
@@ -145,7 +145,7 @@ public function onCommand(CommandSender $sender, Command $cmd, $label, array $ar
                     if($sender instanceof Player){
                         if(count($args) == 2){
                             $fstat = [
-                                'Name' => $args[1],
+                                'Name' => $player->getName(),
```

Comment:
@SalmonGER Salami nab. Add support for show all players one floating stats. Not for one player :/